tests/integration/test_fim/test_files/test_audit_buffer/test_audit_buffer_behavoir.py

```diff
@@ -0,0 +1,311 @@
+'''
+copyright: Copyright (C) 2015-2023, Wazuh Inc.
+
+           Created by Wazuh, Inc. <info@wazuh.com>.
+
+           This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+
+type: integration
+
+brief: File Integrity Monitoring (FIM) system watches selected files and triggering alerts when these files are
+       added, modified or deleted. It can monitor using Audit information (whodata mode). Whodata mode has an option
+       'queue_size' that will save whodata events up until it is full so it can decode them and generate alerts. Events
+       in excess of the queue will be dropped and handled in the next scheduled scan. This is done to avoid blocking
+       the audit socket. This tests aim to test the behavior of the queue, by inserting files above or below the set
+       value for queue_size, and verify if events were detected in whodata or scheduled mode appropiately.
+
+components:
+    - fim
+
+suite: audit_buffer
+
+targets:
+    - agent
+
+daemons:
+    - wazuh-syscheckd
+
+os_platform:
+    - windows
+
+os_version:
+    - Arch Linux
+    - Amazon Linux 2
+    - Amazon Linux 1
+    - CentOS 8
+    - CentOS 7
+    - Debian Buster
+    - Red Hat 8
+    - Ubuntu Focal
+    - Ubuntu Bionic
```

Comment:
Update it, please.

Reply:
Done in https://github.com/wazuh/wazuh-qa/pull/3990/commits/2eb80e3b64add4a8e00deb846a1d2a19a8ab4a09